### ISSUE TITLE
Branch protection on master

### ISSUE THREAD

**Issue by pbugnion** [facultyai/dash-bootstrap-components#112]
When trying to release 0.2.6, I noticed that our release scripts were broken because the master branch was protected. At the moment, our release script:
- pushes a commit and a tag to master as part of `invoke release 0.2.6`
- pushes a commit as part of `invoke postrelease 0.2.6`.

I temporarily removed the restriction, pushed and re-enabled it afterwards. 

I'm not sure who enabled the branch restriction. To fix this permanently, we can do one of:
- remove the branch restriction permanently. This may be OK in the short term, though not being able to push directly to master is probably a good long-term thing. The more contributors we have, the more likely this is to be true.
- fix the build script to push to branches that we merge manually.
- fix the build script to automatically disable the branch restriction on master and re-enable it afterwards. I'm not sure this is even possible, and if it is, it'll probably require GitHub tokens, which adds another layer of complexity to the build process.

**Comment by tcbegley** [comment]
The second of the three options seems like it’s probably the best to me? Not sure why the master branch permissions changed, but I agree it’s probably best to not allow pushing directly to master.

**Comment by srstevenson** [comment]
This was me: when I changed the contact email address for all our open source projects to `opensource@` I disabled branch protection on `master` where it was active so I could commit the change directly using the GitHub editor. Yesterday I remembered I'd done this, and re-enabled it where I thought it had been active, and mistakenly thought that included this repository. Sorry for breaking things.
